tezos/messages/src/p2p/encoding/current_branch.rs

```diff
@@ -16,7 +16,7 @@ use crate::p2p::encoding::block_header::BlockHeader;
 
 use super::limits::CURRENT_BRANCH_HISTORY_MAX_LENGTH;
 
-pub const HISTORY_MAX_SIZE: u8 = CURRENT_BRANCH_HISTORY_MAX_LENGTH as u8; // 200
+pub const HISTORY_MAX_SIZE: u8 = 200;
```

Comment:
if this is supposed to match `CURRENT_BRANCH_HISTORY_MAX_LENGTH`, can we add a static assert to verify that it remains true all the time?

I use this crate in ocaml-interop for that https://docs.rs/static_assertions/1.1.0/static_assertions/index.html

Reply:
Oh wait, I just saw the new value for `CURRENT_BRANCH_HISTORY_MAX_LENGTH`, ignore this comment.